src/Microsoft.Tye.Core/CombineStep.cs

```diff
@@ -28,6 +28,7 @@ public override Task ExecuteAsync(OutputContext output, Application application,
                     var port = binding.Port ?? 80;
                     var urls = $"http://*{(port == 80 ? "" : (":" + port.ToString()))}";
                     service.Service.Environment.Add("ASPNETCORE_URLS", urls);
+                    service.Service.Environment.Add("PORT", port);
```

Comment:
Nice. Is the idea that this is the way to get the listening port for any non-asp.net tech?

Reply:
Yap